utils/grpc_graceful_termination.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/tron-us/go-common/v2/log"
```

Comment:
need a separate line here between tron-us packages and third-party packages

Reply:
@wxue Done. 
Also 
1. Converted methods to functions. 
2. Removed struct type as it was not needed. 
3. Graceful Termination timeout has been added. If it takes more than 10 seconds, we stop the server.

Reply:
LGTM, thanks.